test/oauth-authorization-test.js

```diff
@@ -335,24 +351,38 @@ suite.addBatch({
                         requestToken(cl, this);
                     },
                     function(err, rt) {
+                        var self = this;
+
                         if (err) throw err;
-                        br = new Browser({runScripts: false});
-                        br.visit("http://localhost:4815/oauth/authorize?oauth_token=" + rt.token, this);
+                        br = new Browser({runScripts: false}); // faster without scripts...
+                        Step(
+                            function() {
+                                br.visit("http://localhost:4815/oauth/authorize?oauth_token=" + rt.token, this);
+                            },
+                            function() {
+                                self(!br.success, br);
+                            }
+                        );
                     },
                     function(err, br) {
+                        var self = this;
```

Comment:
In the last function as callback
br.visit("bla",this)... calls next function in Step.     `self(!br.success, br);` calls outer Step